metrics: update docs (#4769)
pingcap/docs: grafana-tidb-dashboard.md

        - Because TiDB supports Multi-Query, it supports sending multiple SQL statements at one time, such as `select 1; select 1; select 1;`. In this case, the total execution time of this query includes the execution time of all SQL statements
    - Command Per Second: the number of commands processed by TiDB per second, which is classified according to the success or failure of command execution results
    - QPS: the number of SQL statements executed per second on all TiDB instances, which is counted according to `SELECT`, `INSERT`, `UPDATE`, and other types of statements
    - CPS By Instance: the command statistics on each TiDB instance, which is classified according to the success or failure of command execution results
    - Failed Query OPM: the statistics of error types (such as syntax errors and primary key conflicts) according to the errors occurred when executing SQL statements per second on each TiDB instance. It contains the module in which the error occurs and the error code

>     - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling). Slow queries are classified into internal and general SQL statements.

Reviewer: ```suggestion
    - Slow query: the statistics of the processing time of slow queries (the time cost of the entire slow query, the time cost of Coprocessor，and the waiting time for Coprocessor scheduling). Slow queries are classified into internal and general SQL statements
```

>     - Connection Idle Duration: duration of idle connection

Reviewer: ```suggestion
    - Connection Idle Duration: the duration of idle connections
```

    - CPU Usage: the statistics of CPU usage of each TiDB instance
    - Connection Count: the number of clients connected to each TiDB instance
    - Open FD Count: the statistics of opened file descriptors of each TiDB instance
    - Disconnection Count: the number of clients disconnected to each TiDB instance
    - Events OPM: the statistics of key events, such as "start", "close", "graceful-shutdown","kill", "hang", and so on
    - Goroutine Count: the number of Goroutines on each TiDB instance
    - Prepare Statement Count: the number of `Prepare` statements that are executed on each TiDB instance and the total count of them
    - Keep Alive OPM: the number of times that the metrics are refreshed every minute on each TiDB instance. It usually needs no attention.

>     - Panic And Critical Error: the number that Panic and Critical Errors occur in TiDB

Reviewer: ```suggestion
    - Panic And Critical Error: the number of panics and critical errors occurred in TiDB
```

    - Time Jump Back OPS: the number of times that the operating system rewinds every second on each TiDB instance
    - Get Token Duration: the time cost of getting Token on each connection

>     - Skip Binlog Count: the number of TiDB failing to write into Binlog

Reviewer: ```suggestion
    - Skip Binlog Count: the number of binlog write failures in TiDB
```
